src/languageSupport/languages/flux/lsp/utils.ts

```diff
@@ -71,23 +71,38 @@ export interface ExecuteCommandInjectTagValue extends ExecuteCommandInjectTag {
 
 export type ExecuteCommandInjectField = ExecuteCommandInjectMeasurement
 
-export interface ExecuteCommandCompositionInit {
+export interface CompositionInitParams {
   bucket: string
   measurement?: string
 }
 
+interface CompositionValueParams extends ExecuteCommandParams {
```

Comment:
I realize it's not really a habit we're in right now, and that these specific interfaces aren't really that complex, but it'd be _really_ helpful if these interfaces (and everything, really) came with some information (in the form of doc comments or whatever) about how they are used and _why_ they are used.

Reply:
absolutely agreed. I can add this in the code cleanup PR after the demo tmrw.
(If I make changes, then I cannot merge. So since the only changes are for code comments, I'll do that tmrw in another PR.)